dist/tools/codespell/ignored_words.txt

```diff
@@ -119,3 +119,6 @@ crate
 
 # VAs (volt ampere second) => was
 vas
+
+# Sur (macOS version alias) => Sure, sir
+Sur
```

Comment:
How the hell can `codespell`'s ignore list be extended if `codespell` complains about the extension itself?

Reply:
Maybe adding a word only works if it is lowercase?

Reply:
Thanks, that seems to solve the problem.